docker/arm-server/Dockerfile

```diff
@@ -40,6 +40,8 @@ RUN apt install -y -f \
 
 RUN echo "num_cvd_accounts=100" >> /etc/default/cuttlefish-host-resources
 
+RUN sed -i s/.*orchestrator_cvd_creation_timeout.*/orchestrator_cvd_creation_timeout=30/g /etc/default/cuttlefish-host_orchestrator
```

Comment:
If cvd_creation_timeout value in `/etc/default/cuttlefish-host_orchestrator` is set as commented, couldn't we simply use `echo` then, not applying regex?

Reply:
Using `echo` seems to be clearer solution than using `sed` changed to use `echo`.